src/model/LabyrinthMatrix.java

```diff
@@ -1,91 +0,0 @@
-package model;
-
-import java.io.File;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthMatrix extends LabyrinthImpl {
-
-	private int matrix[][];
-
-	public LabyrinthMatrix(String path) {
-		Scanner sc;
-		try {
-			sc = new Scanner(new File(path));
-			startValue = sc.nextInt();
-			finishValue = sc.nextInt();
-			freeValue = sc.nextInt();
-			wallValue = sc.nextInt();
-
-			rows = sc.nextInt();
-			columns = sc.nextInt();
-			matrix = new int[rows][columns];
-			for (int i = 0; i < rows; i++)
-				for (int j = 0; j < columns; j++) {
-					matrix[i][j] = sc.nextInt();
-					if (matrix[i][j] == startValue)
-						start = new Position(i, j);
-					if (matrix[i][j] == finishValue)
-						finish = new Position(i, j);
-				}
-			sc.close();
-
-		} catch (Exception ex) {
```

Comment:
Is is not very good to catch excepltion. Example can be if the file is bad formatter(contains unwanted values, strings instead of number etc). the exception you get is that the file is not found that is not true.